src/bundle/DependencyInjection/ContainerBuilder.php

```diff
@@ -59,4 +64,73 @@ private function resolveExtensionName(string $name): string
 
         return self::$extensionNameMap[$name] ?? $name;
     }
+
+    public function getDefinition(string $id)
+    {
+        if (!isset(self::$serviceNameMap)) {
+            /** @noinspection PhpIncludeInspection */
+            self::$serviceNameMap = require IbexaCompatibilityLayerBundle::MAPPINGS_PATH
+                . \DIRECTORY_SEPARATOR . 'services-to-fqcn-map.php';
+        }
+
+        try {
+            return parent::getDefinition($id);
+        } catch (ServiceNotFoundException $e) {
+            if (isset(self::$serviceNameMap[$id])) {
+                return parent::getDefinition(self::$serviceNameMap[$id]);
+            }
+        }
+
+        throw $e;
+    }
+
+    public function hasDefinition(string $id)
+    {
+        if (!isset(self::$serviceNameMap)) {
+            /** @noinspection PhpIncludeInspection */
+            self::$serviceNameMap = require IbexaCompatibilityLayerBundle::MAPPINGS_PATH
+                . \DIRECTORY_SEPARATOR . 'services-to-fqcn-map.php';
+        }
+
+        $hasDefinition = parent::hasDefinition($id);
+        if (!$hasDefinition && isset(self::$serviceNameMap[$id])) {
+            return parent::hasDefinition(self::$serviceNameMap[$id]);
+        }
+        return $hasDefinition;
```

Comment:
CS: missing new line. I think for this repository you need to manually run `composer check-cs`, since there's no CI doing that.